lago/plugins/vm.py

```diff
@@ -360,7 +360,7 @@ def wait_for_ssh(self):
         return ssh.wait_for_ssh(
             ip_addr=self.ip(),
             host_name=self.name(),
-            connect_retries=self._spec.get('boot_time_sec', 50),
+            connect_timeout=self._spec.get('boot_time_sec', 600),
```

Comment:
The idea here isn't about making the ssh timeout longer. (I'm familiar with
the conf file )
As I was written in the PR, when the host in is booting and got it's IP and
didn't open port 22 or get sshd up and running - it will block any ssh
communication from Lago . so, no matter if I will raise the ssh timeout to
1M secs , Lago won't wait for the ssh connection the time that configured -
it will stop waiting and try again. In this situation we loose 1 retry in
one second instead of the time configured (10 sec at default).

My suggestion is ,wait for the connection to the host a specific amount of
time (say 10 minutes), at that time, Lago will try to connect to the host
(with an interval of 1 sec like now) as many times that it can.

On Wed, Aug 10, 2016 at 9:21 PM, David Caro <notifications@github.com>
wrote:

> In lago/plugins/vm.py
> <https://github.com/lago-project/lago/pull/287#discussion_r74301014>:
>
> > @@ -360,7 +360,7 @@ def wait_for_ssh(self):
> >          return ssh.wait_for_ssh(
> >              ip_addr=self.ip(),
> >              host_name=self.name(),
> > -            connect_retries=self._spec.get('boot_time_sec', 50),
> > +            connect_timeout=self._spec.get('boot_time_sec', 600),
>
> To tweak the ssh timeout you have the ssh_timeout conf value:
>
> https://github.com/lago-project/lago/pull/287/files#diff-
> 031917605b8ea7e9f49d56da61e00f6eR319
>
> Not sure if that's what you wanted here.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lago-project/lago/pull/287/files/da8df216ddf1306dca0e29cdd2f729cd5f2da8f3#r74301014>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARdD8kQPKOznvQBPU-i3JA2W6-jdhOe7ks5qehatgaJpZM4I7jSN>
> .
>